dash-pipeline/bmv2/dash_dpdk_pna.p4

```diff
@@ -0,0 +1,16 @@
+
+control dash_precontrol(
+    in    headers_t pre_hdr,
+    inout metadata_t pre_user_meta,
+    in    pna_pre_input_metadata_t  istd,
+    inout pna_pre_output_metadata_t ostd)
+{
+    apply {
+    }
+}
+
+PNA_NIC(
+    dash_parser(),
+    dash_precontrol(),
+    dash_ingress(),
+    dash_deparser()) main;
```

Comment:
Since PNA is a "portable" architecture, why tie it to DPDK only (that is, why call this file dpdk_pna.p4).  We can always pick different targets.  Isn't it?  Putting portable architectures with the targets just defeats the purpose in my view.  This is true towards usage of DPDK target with PNA name elsewhere in this PR as well.

Reply:
I am happy to rename the file "dash_pna.p4" if you and other prefer that file name, and I can change the preprocessor symbol to ARCH_PNA instead of ARCH_DPDK_PNA, too.

Changing those things will not automatically mean that this code compiles on all PNA target devices, since this code is written to be a functional reference model for a CPU-based target.  Different PNA devices will have different subsets of possible PNA programs they can compile, and this program seems unlikely to be in the intersection of all of those.